### ISSUE TITLE
Implement a page for initial user setup in the Config UI

### ISSUE THREAD

**Issue by ThomasLaPiana** [ethyca/fides#567]
### Is your feature request related to a specific problem?

We need a UI page for creating users, specifically for setting up an initial admin user

### Describe the solution you'd like

A minimalist form page that collects user information and sends it to the server for creation.

### Describe alternatives you've considered, if any

_A description of any alternative solutions or features you've considered._

### Additional context

* username is an email
* password must be entered twice for confirmation
* provide link to fides "help" docs

**Comment by LKCSmith** [comment]
Points: 1 but I think this is blocked until some BE work is completed?

**Comment by ThomasLaPiana** [comment]
@LKCSmith yep! We're waiting to get the user management (fideslib) code integrated before we can ship users-related things

**Comment by LKCSmith** [comment]
@mfbrown - I don't think the BE work is yet completed for this to be implemented yet.

**Comment by ThomasLaPiana** [comment]
correct, still waiting on the fideslib user management but its coming soon ™️

**Comment by ThomasLaPiana** [comment]
closing, completed via another ticket
